action-server/actions/answers.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import BaseModel
+from typing import List
+
+import requests
+
+ANSWERS_PATH = "answers"
+QUESTION_KEY = "question"
+
+
+class QaResponse(BaseModel):
+    answers: List[str]
+    status_code: int
```

Comment:
It's the HTTP status code?

Reply:
Maybe make it more explicit? We might have some kind of  status from the API itself.

Reply:
Yes I thought the form could receive more of a pre-digested status merging status code and the API status. Something like "success" and "error", to which we could add the "need_assessment" and "out_of_distribution" options later.

Reply:
`status_code` -> `http_status_code`

Reply:
After Karine's comment, I've put in an Enum for the query status that only contains `Success` or `Failure` for now, depending on the http status code.